test/e2e/ipfs-companion.test.js

```diff
@@ -0,0 +1,157 @@
+import { Builder, By, Key } from 'selenium-webdriver'
+import { describe, it, before } from 'mocha'
+import { expect } from 'chai'
+import fs from 'fs'
+import chrome from 'selenium-webdriver/chrome.js'
+import firefox from 'selenium-webdriver/firefox.js'
+
+async function delay (ms) {
+  return new Promise(res => setTimeout(res, ms), _ => {})
+}
+
+function getVersion () {
+  return process.env.IPFS_COMPANION_VERSION || JSON.parse(fs.readFileSync('add-on/manifest.common.json')).version
+}
+
+async function openChromiumBrowser () {
+  console.info('Opening Chromium browser')
+  const extension = `build/ipfs_companion-${getVersion()}_chromium.zip`
+  console.info(`Checking if ${extension} exists`)
+  expect(fs.existsSync(extension)).to.be.true // eslint-disable-line no-unused-expressions
+  const options = new chrome.Options()
+  options.addExtensions(extension)
+  options.addArguments('--lang=en-GB,en-US')
+  if (process.env.TEST_HEADLESS === '1') {
+    options.addArguments('--headless=chrome')
+  }
+  const builder = new Builder()
+    .forBrowser('chrome')
+    .setChromeOptions(options)
+  if (process.env.SELENIUM_REMOTE_CHROMIUM_URL !== undefined) {
```

Comment:
I don't quite understand this one. Also, FYI, the linter would complain about `!=` not being `!==`.

Reply:
@galargh linter should not complain about `!= null`. If it is, then the type is overloaded or something else is off, but ditto to what Nishant said. 

I'm fine with `!== undefined` for env vars because they're either strings or undefined, but I prefer `!= null` as a general "maintainable code" practice for communicating to future devs (myself) that it doesn't matter that it's not explicitly undefined, only that it's defined. But again, for env vars, the nuance here is tiny, if not completely non-existent.